.github/workflows/lint-and-tests.yml

```diff
@@ -28,26 +28,11 @@ permissions:
   # This permission is required by `MishaKav/jest-coverage-comment`
   pull-requests: write
 
-jobs:
-  base:
-    name: Base Tasks
-    runs-on: ubuntu-latest
-    outputs:
-      turbo_args: ${{ steps.turborepo_arguments.outputs.turbo_args }}
-
-    steps:
-      - name: Harden Runner
-        uses: step-security/harden-runner@4d991eb9b905ef189e4c376166672c3f2f230481 # v2.11.0
-        with:
-          egress-policy: audit
-
-      - name: Provide Turborepo Arguments
-        # This step is responsible for providing a reusable string that can be used within other steps and jobs
-        # that use the `turbo` cli command as a way of easily providing shared arguments to the `turbo` command
-        id: turborepo_arguments
-        # See https://turbo.build/repo/docs/reference/command-line-reference/run#--force
-        run: echo "turbo_args=--force=true" >> "$GITHUB_OUTPUT"
+env:
+  # See https://turbo.build/repo/docs/reference/command-line-reference/run#--force
+  TURBO_ARGS: --force
```

Comment:
Should we just set `TURBO_FORCE` instead of needing to pass these args into each turbo call, or are there other turbo calls happening that don't want this?